test/fixtures/customErrorPlugin.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+let DescribedMixin
+let DeserializedMixin
+let ObjectValue
+
+class CustomError extends Error {
+  constructor (message, code) {
+    super(message)
+    this.code = code
+    this.name = 'CustomError'
+  }
+}
+
+exports.CustomError = CustomError
+
+exports.setDependencies = function (dependencies) {
```

Comment:
I think it'd be better to follow the `register` pattern used here:

https://github.com/concordancejs/react/blob/cc84400d6f805fd57338f4101abd4e30d43f3508/index.js#L59

Reply:
Done